pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -24,41 +24,36 @@
 # Usually, this option is broken, do not use it except if you know what you are
 # doing.
 , sourceDebug ? false
+, projectDscPath ? {
+    i686 = "OvmfPkg/OvmfPkgIa32.dsc";
+    x86_64 = "OvmfPkg/OvmfPkgX64.dsc";
+    aarch64 = "ArmVirtPkg/ArmVirtQemu.dsc";
+    riscv64 = "OvmfPkg/RiscVVirt/RiscVVirtQemu.dsc";
+  }.${stdenv.hostPlatform.parsed.cpu.name}
```

Comment:
This will explode at evaluation with an unexpected hostPlatform, you need a `or throw "Unsupported ..."`, failure to attribute select is an abort in Nix.

Reply:
Please also mention that this is in OVMF so that end users will know where this throw comes from.

Reply:
Good ideas. Thanks!